frontend/src/components/MainNavigation.vue

```diff
@@ -429,6 +496,40 @@ export default {
         }
       }
     }
+
+    .project-menu {
+      border-radius: 0;
+
+      >>>.v-card {
```

Comment:
fragile

Reply:
True, but actually old code that I moved from non-scoped css to scoped. I added tests that analyze the class hierarchy and (hopefully) detect incompatible API changes. Of course, this will probably break with vuetify 2 anyway but then we will have to deal with it and adapt accordingly.